.github/workflows/links.yml

```diff
@@ -38,3 +38,12 @@ jobs:
           title: Link Checker Report
           content-filepath: ./lychee/out.md
           labels: report, automated issue
+
+      - name: Notify in Slack in case of failure
+        id: slack-notification
+        if: ${{ github.event_name == 'schedule' && env.lychee_exit_code != 0 }}
```

Comment:
I'm wondering what's the difference to `failure() && ... schedule` instead?
Feels better aligned than checking the error code of lynchee